### PR TITLE
refactor(typography): align with current style guides

### LINKED ISSUE #263: refactor(breadcrumb): follow current style guides and fix typescript issues
components is not currently following our current style guides and tsc is returning following errors

```
9 export class BreadcrumbItem extends Component<BreadcrumbItemProps> {
                                                ~~~~~~~~~~~~~~~~~~~

Breadcrumb-item.tsx(9,47): error TS4020: Extends clause of exported class 'BreadcrumbItem' has or is using private name 'BreadcrumbItemProps'.


13   static get props() {
                ~~~~~

Breadcrumb-item.tsx(13,14): error TS4038: Return type of public static property getter from exported class has or is using name 'PropOptions' from external module "/Users/mario/ngParty/blaze-elements/node_modules/skatejs/src/ts-typings/api" but cannot be named.


7 export class Breadcrumb extends Component<BreadcrumbProps> {
                                            ~~~~~~~~~~~~~~~

Breadcrumb.tsx(7,43): error TS4020: Extends clause of exported class 'Breadcrumb' has or is using private name 'BreadcrumbProps'.
```

### LINKED ISSUE #282: refactor(typography): follow current style guides and fix typescript issues
components is not currently following our current style guides and tsc is returning following errors

```

10 export class Abbreviation extends Component<AbbreviationProps> {
                                               ~~~~~~~~~~~~~~~~~

Abbreviation.tsx(10,45): error TS4020: Extends clause of exported class 'Abbreviation' has or is using private name 'AbbreviationProps'.


13   static get props() {
                ~~~~~

Abbreviation.tsx(13,14): error TS4038: Return type of public static property getter from exported class has or is using name 'PropOptions' from external module "/Users/mario/ngParty/blaze-elements/node_modules/skatejs/src/ts-typings/api" but cannot be named.


10 export class Blockquote extends Component<BlockquoteProps> {
                                             ~~~~~~~~~~~~~~~

Blockquote.tsx(10,43): error TS4020: Extends clause of exported class 'Blockquote' has or is using private name 'BlockquoteProps'.


12   static get props() {
                ~~~~~

Blockquote.tsx(12,14): error TS4038: Return type of public static property getter from exported class has or is using name 'PropOptions' from external module "/Users/mario/ngParty/blaze-elements/node_modules/skatejs/src/ts-typings/api" but cannot be named.


10 export class Code extends Component<CodeProps> {
                                       ~~~~~~~~~

Code.tsx(10,37): error TS4020: Extends clause of exported class 'Code' has or is using private name 'CodeProps'.


13   static get props() {
                ~~~~~

Code.tsx(13,14): error TS4038: Return type of public static property getter from exported class has or is using name 'PropOptions' from external module "/Users/mario/ngParty/blaze-elements/node_modules/skatejs/src/ts-typings/api" but cannot be named.


9 export class HighlightedText extends Component<HighlightedTextProps> {
                                                 ~~~~~~~~~~~~~~~~~~~~

HighlightedText.tsx(9,48): error TS4020: Extends clause of exported class 'HighlightedText' has or is using private name 'HighlightedTextProps'.


9 export class KeyboardKeys extends Component<KeyboardKeysProps> {
                                              ~~~~~~~~~~~~~~~~~

KeyboardKeys.tsx(9,45): error TS4020: Extends clause of exported class 'KeyboardKeys' has or is using private name 'KeyboardKeysProps'.


22 export class Link extends Component<LinkProps> {
                                       ~~~~~~~~~

Link.tsx(22,37): error TS4020: Extends clause of exported class 'Link' has or is using private name 'LinkProps'.


26   static get props() {
                ~~~~~

Link.tsx(26,14): error TS4038: Return type of public static property getter from exported class has or is using name 'PropOptions' from external module "/Users/mario/ngParty/blaze-elements/node_modules/skatejs/src/ts-typings/api" but cannot be named.


9 export class LoudText extends Component<LoudTextProps> {
                                          ~~~~~~~~~~~~~

LoudText.tsx(9,41): error TS4020: Extends clause of exported class 'LoudText' has or is using private name 'LoudTextProps'.


9 export class MonospaceText extends Component<MonospaceTextProps> {
                                               ~~~~~~~~~~~~~~~~~~

MonospaceText.tsx(9,46): error TS4020: Extends clause of exported class 'MonospaceText' has or is using private name 'MonospaceTextProps'.


9 export class Paragraph extends Component<ParagraphProps> {
                                           ~~~~~~~~~~~~~~

Paragraph.tsx(9,42): error TS4020: Extends clause of exported class 'Paragraph' has or is using private name 'ParagraphProps'.


9 export class QuietText extends Component<QuietTextProps> {
                                           ~~~~~~~~~~~~~~

QuietText.tsx(9,42): error TS4020: Extends clause of exported class 'QuietText' has or is using private name 'QuietTextProps'.

```

### PR DESCRIPTION
**Pull request by devopsmariocom**
affects: @blaze-elements/typography

Closes #282

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/wc-catalogue/blaze-elements/blob/master/docs/CONTRIBUTING.md#commit
- [x] There are no linting errors and code is properly formatted (your run before push `yarn ts:format && yarn ts:lint:fix`)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: packages/typography/Abbreviation.tsx [hunk 1]
```diff
@@ -1,39 +1,36 @@
...
+};
+
+export type EventProps = {};
+
+export type AbbreviationEvents = {};
+
+@shadyCssStyles()
+export default class Abbreviation extends Component<AbbreviationProps> {
 
+  // FIX: enable projection from attribute
```
Comment (devopsmariocom) [inline packages/typography/Abbreviation.tsx]: This is same issue as in https://github.com/wc-catalogue/blaze-elements/pull/275#discussion_r105011772

**Comment by davidrus** [comment]
Please change issue number in commit message to #282 , current issue number is for breadcrumb :)

File: packages/typography/Abbreviation.tsx (hunk 1 above)
Comment (davidrus) [reply packages/typography/Abbreviation.tsx]: As we talk about this, please try use skate prop to define this property.

**Review by davidrus — CHANGES_REQUESTED** [review]
Good job 👍 , just change closes issue number and try prop from skate.

**Review by davidrus — APPROVED** [review]

**Pull request merged**